app/src/controllers/table.js

```diff
@@ -0,0 +1,194 @@
+(function() {
+  'use strict';
+  angular.module('prodocs.controllers')
+    .controller('tableCtrl', ['$scope', '$mdMedia',
+      '$mdSidenav', '$timeout', 'Utils', 'Docs',
+      function($scope, $mdMedia, $mdSidenav, $timeout, Utils, Docs) {
```

Comment:
'Docs' is defined but never used.<br>'Utils' is defined but never used.<br>'$timeout' is defined but never used.<br>'$mdSidenav' is defined but never used.<br>'$mdMedia' is defined but never used.